uint32 set_pins
uint32 clear_pins
---
bool success

